apps/src/templates/projects/submitProjectDialog/submit-project-dialog.module.scss

```diff
@@ -0,0 +1,67 @@
+@import "color", "font";
+
+.submitProjectDialog {
+  max-height: calc(95vh - 100px);
```

Comment:
Just curious where this value comes from?

Reply:
Ahh... I first used this for the `aichat` warning modal so that the modal can be scrolled for short windows. https://github.com/code-dot-org/code-dot-org/pull/61215 (100px less than 95% of the viewport height).